### PR TITLE
VSCode extension: Improvements to configuration procedure

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 5.3.0

Resolves: https://issues.redhat.com/browse/WINDUP-3369 by making the suggested additions to the run configuration procedure. 

Preview: https://deploy-preview-539--windup-documentation.netlify.app/docs/vs-code-extension-guide/master/index.html#vs-code-extension-run-configuration_vsc-extension-guide

### CODE REVIEW

**Comment by RichardHoch** [comment]
@sbeskin-redhat: Please review.

File: docs/topics/vs-code-extension-run-configuration.adoc [hunk 1]
```diff
@@ -14,7 +14,11 @@ You can configure multiple run configurations to run against each project you im
 . Click the *+* (plus sign) next to *Migration Toolkit for Applications* to add a run configuration.
 . Complete the following configuration fields:
 
-** *Input*: Click *Add* and do one of the following:
+** *Name*: Enter a meaningful name for the analysis configuration or accept the default.
+** *`mta-cli`*: Enter the path to the `mta-cli` executable. For example: `$HOME/mta-cli-5.3.0.Final/bin/mta-cli`.
++
+You can download the `mta-cli` executable from link:https://developers.redhat.com/products/mta/download[MTA download].
```
Comment (sbeskin-redhat) [inline docs/topics/vs-code-extension-run-configuration.adoc:20]: @RichardHoch 
It seems to me that having MTA CLI executable should be a prerequisite for this procedure. Something like "You need to have the MTA CLI executable available. If you do not have it,  download it from ..."

File: docs/topics/vs-code-extension-run-configuration.adoc [hunk 2]
```diff
@@ -14,7 +14,11 @@ You can configure multiple run configurations to run against each project you im
 . Click the *+* (plus sign) next to *Migration Toolkit for Applications* to add a run configuration.
 . Complete the following configuration fields:
 
-** *Input*: Click *Add* and do one of the following:
+** *Name*: Enter a meaningful name for the analysis configuration or accept the default.
```
Comment (sbeskin-redhat) [inline docs/topics/vs-code-extension-run-configuration.adoc:17]: @RichardHoch 
Does the default name appear in the Name field?

Comment (RichardHoch) [reply docs/topics/vs-code-extension-run-configuration.adoc:17]: @sbeskin-redhat Yes, it does

File: docs/topics/vs-code-extension-run-configuration.adoc (hunk 1 above)
Comment (RichardHoch) [reply docs/topics/vs-code-extension-run-configuration.adoc:20]: @sbeskin-redhat Good point. I made it a prerequisite.

**Comment by RichardHoch** [comment]
@sbeskin-redhat Please review again.

File: docs/topics/vs-code-extension-run-configuration.adoc (hunk 2 above)
Comment (sbeskin-redhat) [reply docs/topics/vs-code-extension-run-configuration.adoc:21]: @RichardHoch 
No, it's fine as is. I just wanted to make sure that the default name is already there.

File: docs/topics/vs-code-extension-run-configuration.adoc [hunk 3]
```diff
@@ -8,13 +8,19 @@
 
 You can configure multiple run configurations to run against each project you import to VS Code.
 
+.Prerequisites
+
+* `mta-cli` execrable installed. You can download the `mta-cli` executable from link:https://developers.redhat.com/products/mta/download[MTA download].
+
```
Comment (sbeskin-redhat) [inline docs/topics/vs-code-extension-run-configuration.adoc:14]: @RichardHoch 
executable

Comment (RichardHoch) [reply docs/topics/vs-code-extension-run-configuration.adoc:14]: Fixed.

Comment (sbeskin-redhat) [reply docs/topics/vs-code-extension-run-configuration.adoc:14]: Looks OK

**Pull request merged**